CnCNet/Net/Tunnel/TunnelClient.cs

```diff
@@ -1,29 +1,25 @@
-﻿using System;
-using System.Collections.Generic;
-using System.Linq;
-using System.Text;
+﻿namespace CnCNetServer;
+
 using System.Net;
 
-namespace CnCNet.Net.Tunnel
+internal sealed class TunnelClient
 {
-    class TunnelClient
-    {
-        public IPEndPoint RemoteEP;
+    private readonly int timeout;
+
+    private long lastReceiveTick;
 
-        long LastReceiveTick;
-        int Timeout;
+    public TunnelClient(IPEndPoint remoteEndPoint, int timeout = 30)
+    {
+        RemoteEp = remoteEndPoint;
+        this.timeout = timeout;
+    }
 
-        public TunnelClient(int timeout = 30)
-        {
-            Timeout = timeout;
-        }
+    public IPEndPoint RemoteEp { get; set; }
 
-        public bool TimedOut {
-            get { return TimeSpan.FromTicks(DateTime.UtcNow.Ticks - LastReceiveTick).TotalSeconds >= Timeout; } }
+    public bool TimedOut { get => TimeSpan.FromTicks(DateTime.UtcNow.Ticks - lastReceiveTick).TotalSeconds >= timeout; }
 
-        public void SetLastReceiveTick()
-        {
-            LastReceiveTick = DateTime.UtcNow.Ticks;
-        }
+    public void SetLastReceiveTick()
+    {
+        lastReceiveTick = DateTime.UtcNow.Ticks;
     }
```

Comment:
We generally use expression bodies for one-line functions:
```suggestion
    public void SetLastReceiveTick()
        => lastReceiveTick = DateTime.UtcNow.Ticks;
```
It would be good if you went over the code and updated it to match.